### DISCUSSION TITLE
Draw.io web conversion options from Visio

### DISCUSSION THREAD

**Discussion by eabad2** [jgraph/drawio-desktop#1839] [General]
* [x ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
We have a large team within the Enterprise Technical Infrastructure Platforms Services team that currently uses Visio, which will be deprecated April 2025.  They are looking to move off of Visio and to use Draw.io desktop product as a replacement.  
We have specific questions about the web converter that is currently used by [draw.io].  Our understanding is that they do have a converter, however, the data will need to flow to their site to convert, which poses a security issue on our end.  

Is it possible to acquire the ability to perform [VSD conversions](https://www.drawio.com/doc/faq/vsd-import-desktop) outside of the drawings.net site?  Interested in funding features, contributions, etc.  I have a large user group > 10K+ using the draw.io desktop app.  Example: could I have an integration / plugin in desktop that handle VSD conversion against my private service.

**Describe the solution you'd like**
Is it possible to acquire the ability to perform [VSD conversions](https://www.drawio.com/doc/faq/vsd-import-desktop) outside of the drawings.net site?  Interested in funding features, contributions, etc.  I have a large user group > 10K+ using the draw.io desktop app.  Example: could I have an integration / plugin in desktop that handle VSD conversion against my private service.

**Describe alternatives you've considered**
Looking for another alternative product that can perform conversions on-prem vs. cloud or funding feature option.

**Additional context**

**Comment by davidjgraph** [comment]
Create a powershell script that converts all .vsd files to .vsdx.
